src/Package/Impl/Repl/VsRInteractiveWorkflowProvider.cs

```diff
@@ -49,14 +49,21 @@ internal class VsRInteractiveWorkflowProvider : IRInteractiveWorkflowProvider {
             _shell = shell;
         }
 
+        public void Dispose() {
+            if (_instanceLazy?.IsValueCreated == true) {
```

Comment:
I'll use DisposableBag instead, and add the workflow to it from CreateRInteractiveFlow. 

---
In reply to: [74974926](https://github.com/Microsoft/RTVS/pull/2173#discussion_r74974926) [](ancestors = 74974926)